mcux/middleware/mcux-sdk-middleware-connectivity-framework/SW-Content-Register.txt

```diff
@@ -0,0 +1,12 @@
+Release Name: MCUXpresso Software Development Kit (SDK) Framework Middleware
+Release Version: 6.1.0
+Package License: LA_OPT_NXP_Software_License.htm - Additional Distribution License, Section 2.3 applies  
```

Comment:
Is this framework licensed under LA_OPT, or BSD-3? All the files seem licensed under BSD-3, but the LA_OPT license is pulled in and referenced here. The LA_OPT license isn't compatible with Zephyr, so we need to clear this up

Reply:
I think we can remove it as all files are under BSD indeed, @yeaissa will address.
FYI, mcux-sdk folder seems to contain the LA_OPT license which is probably problematic according to your comment:
https://github.com/zephyrproject-rtos/hal_nxp/tree/master/mcux/mcux-sdk

Reply:
Thanks for the heads up- it is probably problematic, yes. I'll discuss this internally and see what we can do there.

Reply:
After checking with @yvanderv, I've confirmed we should drop this file from this PR as well as the MCUX SDK side